edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Http/controllers/RegistryController.cs

```diff
@@ -517,11 +517,11 @@ Task SendErrorResponse(Exception ex)
         {
             if (ex is ValidationException ve)
             {
-                return this.SendResponseAsync(ve.StatusCode, ve.Message);
+                return this.SendResponseAsync(ve.StatusCode, FormatErrorResponseMessage(ve.Message));
             }
             else
             {
-                return this.SendResponseAsync(HttpStatusCode.InternalServerError, ex.Message);
+                return this.SendResponseAsync(HttpStatusCode.InternalServerError, FormatErrorResponseMessage(ex.Message));
```

Comment:
Do we need to update any tests that were expecting old format?

Reply:
no, there were no tests expecting the old format